docs/identities/sign-in/saml.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: saml
+title: Login with SAML
+sidebar_label: SAML
+slug: saml
+---
+
+SAML (Security Assertion Markup Language) is an XML-based open standard used for exchanging authentication and authorization data
+between parties.
+
+This guide will walk you through the steps required to set up SAML Single Sign-On (SSO) with Ory Network using BoxyHQ as your SAML
+provider.
+
+## Prerequisites
+
+Before proceeding, ensure you have the following:
+
+- Access to [Ory Network](https://console.ory.sh/)
+- An active account with [BoxyHQ](https://app.eu.boxyhq.com/auth/join)
+- [Ory CLI](../../guides/cli/installation)
+
+:::info
+
+SAML integration in Ory Network uses the [B2B Organization feature](../../kratos/organizations/) which is available on the Ory
+Network Growth plan. [Reach out](https://ory.sh/contact) if you are interested in a trial.
+
+:::
+
+## Configuration
+
+To set up the integration, you'll need to get your Ory Network session token:
+
+- [Install the Ory CLI](../../guides/cli/installation) on your system.
+- Run `ory auth` to sign into your Ory Network account.
+- Locate the session token in the `.ory-cloud.json` file in your home folder. This token starts with the prefix `ory_st`.
```

Comment:
let's put simple `cat dir/file | grep` to make it easier to find ?

Reply:
done